### DISCUSSION TITLE
登录教程

### DISCUSSION THREAD

**Discussion by LuYaoSEN** [share121/inter-knot#5860] [公告]
点击设置，然后给改成用户端口，然后复制进入，然后把那个码填写进去，然后登录就行了
![Screenshot_20241227_181726](https://github.com/user-attachments/assets/00e5496b-2821-43f2-881d-3630d2f4ca61)

**Comment by Bronya-bit** [comment]
温馨提示：未登录看不着 #

**Reply by LuYaoSEN** [reply]
能啊

**Comment by MO-hy** [comment]
没登陆也可以看的，比如我现在就是登录过期的状态

**Comment by MO-hy** [comment]
回错人了哈哈

**Comment by Jaques81** [comment]
突然发现原来是个换皮github

**Comment by AAAElysian** [comment]
什么码啊

**Comment by LIU-145276** [comment]
哈哈哈，原来如此

**Comment by yuyuyubi** [comment]
复制什么啊，进入什么啊，没看懂啊😢

**Reply by Yizakl** [reply]
建议你不要登录了（

**Comment by 208166153Cindy** [comment]
问一下设置在哪没找到

**Reply by 208166153Cindy** [reply]
可以了谢谢

**Comment by Niedhogue999** [comment]
我显示是未登录但是以前登陆过，我用的是公共接口一旦切换成用户接口的话就看不到应该发的东西了就是这样
![Uploading 1737828398922.jpg…]()
